### DISCUSSION TITLE
How kong route by port?

### DISCUSSION THREAD

**Discussion by xlb0479** [Kong/kong#7273] [Ideas and feature requests]
Now I know with proper config, kong can listen on multi port.
But when I create a route, how to route request by port？
Thanks.

**Comment by bungle** [comment]
Routing by `port` is currently only supported by `stream`s, and not with `http/https`. It will be supported in a future.

**Comment by wenerme** [comment]
unifi controller required 8443(factory),really hope this feature implemented
